subsys/nfc/t2t/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NFC_T2T_PARSER
+	bool
+	prompt "NFC Type 2 Tag parser libraries"
```

Comment:
`prompt` is redundant. Place the text next to `bool`. For consistency with other Kconfig files.

Reply:
Ok